src/django_idom/http/views.py

```diff
@@ -6,7 +6,7 @@
 from django.http import HttpRequest, HttpResponse
 from idom.config import IDOM_WED_MODULES_DIR
 
-from .config import IDOM_WEB_MODULE_CACHE, IDOM_WEB_MODULE_LRU_CACHE_SIZE
+from ..config import IDOM_WEB_MODULE_CACHE, IDOM_WEB_MODULE_LRU_CACHE_SIZE
```

Comment:
For my repos I avoid using relative imports, since absolute imports makes it easier for refactoring utilities (ex `rope` and `vulture`) to do their jobs properly.

If I do use relative imports, two levels up is typically the max I'll let slide.

I can change this double relative import to absolute, but should I also to change the handful of relative imports this repo has to absolute?

Reply:
I use single-level relative imports all the time. I personally just avoid double. I'm fine either way though.